pkg/apiserver/api_installer.go

```diff
@@ -517,6 +531,10 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 			namespaced = ""
 		}
 
+		if kubeVerb, found := toDiscoveryKubeVerb[action.Verb]; found {
```

Comment:
I'd rather explicitly map "CONNECT":"" and exclude it so we can fail if we encounter other unknown verbs